src/subworkflows/create.js

```diff
@@ -110,21 +120,28 @@ function createUnit({ config, application, unitBuilders, unitFactoryCls }) {
  * @param application {*} application (optional)
  * @returns {*}
  */
-function createDynamicUnits({ dynamicSubworkflow, units, unitBuilders, unitFactoryCls, application = null }) {
+function createDynamicUnits({
+    dynamicSubworkflow,
+    units,
+    unitBuilders,
+    unitFactoryCls,
+    application = null,
+}) {
     const { name, subfolder } = dynamicSubworkflow;
-    switch(name) {
+    switch (name) {
         case "surfaceEnergy":
-            const [ scfUnit ] = units;
+            // eslint-disable-next-line no-case-declarations
+            const [scfUnit] = units;
             return getSurfaceEnergySubworkflowUnits({ scfUnit, unitBuilders });
         case "getQpointIrrep":
+            // eslint-disable-next-line no-case-declarations
```

Comment:
If we movet he const statements above switch, we can avoid having to override eslint